backend/ordd_api/views.py

```diff
@@ -226,10 +226,11 @@ def get(self, request, *args, **kwargs):
             MAIL_SUBJECT_PREFIX, user.username))
         content = ("""New user '%s' has activated his or her account.<br>
 EMail address: '%s'.<br>""" % (user.username, user.email))
-        mailer(ORDD_ADMIN_MAIL, subject,
-               {"title": subject,
-                "content": content},
-               None, 'base')
+        for admin in Profile.objects.filter(user__groups__name='admin'):
+            mailer(admin.user.email, subject,
```

Comment:
`admin.user.email` should always exist because is required by the signup form, right?